src/js/components/products/list.tsx

```diff
@@ -206,12 +207,12 @@ const ProductsList = () => {
       <>
         <Header history={history} />
         <PageHeader />
-        <div className="slds-p-around_x-large">
+        <div className="slds-text-heading_small slds-p-around_x-large">
           {window.GLOBALS.SITE?.welcome_text ? (
             // These messages are pre-cleaned by the API
             <div
               className="markdown
-                slds-p-bottom_medium
+                slds-text-heading_small
                 slds-text-longform
```

Comment:
Don't these two helper classes conflict with each other?
```suggestion
                slds-text-heading_small
```

Reply:
removed longform